lib/storage/file.js

```diff
@@ -798,6 +798,68 @@ File.prototype.setMetadata = function(metadata, callback) {
 };
 
 /**
+ * Set a file to be private to the project and removes all other permissions.
+ * If you set `options.strict` to true, it will set it to be private to the
+ * owner user only.
```

Comment:
Hope you don't mind these small change suggestions:

```diff
- Set a file to be private to the project and removes all other permissions.
+ Make a file private to the project and remove all other permissions.

- If you set `options.strict` to true, it will set it to be private to the owner user only.
+ Set `options.strict` to true to make the file private to only the owner.
```